src/Installers/Windows/WindowsInstallers.proj

```diff
@@ -19,11 +23,13 @@
     <InstallerProject Include="SharedFrameworkBundle/SharedFrameworkBundle.wixproj" AdditionalProperties="Platform=x64" />
     <InstallerProject Include="SharedFrameworkBundle/SharedFrameworkBundle.wixproj" AdditionalProperties="Platform=x86" />
 
-    <InstallerProject Include="*/*.wixproj" Exclude="@(InstallerProject);ancm/**/*" />
```

Comment:
I think this was covering the SharedFramework and SharedFrameworkLib projects, do we still need this?